spec/requests/articles/articles_destroy_spec.rb

```diff
@@ -14,11 +14,11 @@
     expect(destroyed_article).to be_nil
   end
 
-  it "schedules a RemoveAllJob if there are comments" do
+  it "schedules a RemoveAllWorker if there are comments" do
     create(:comment, commentable: article, user: user)
     expect do
       delete "/articles/#{article.id}"
-    end.to have_enqueued_job(Notifications::RemoveAllJob).once
+    end.to sidekiq_assert_enqueued_jobs(1, only: Notifications::RemoveAllWorker)
```

Comment:
this can't work, you need to pass the block (the delete) inside `sidekiq_assert_enqueued_jobs` and remove the expect